### ISSUE TITLE
InteractiveChat

### ISSUE THREAD

**Issue by KaspianDev** [mfnalex/Community-Survival#7]
This plugin adds json messages (those you can hover on)
Pings with sounds and shit
Item preview in chat
and it supports all chat plugins
https://www.spigotmc.org/resources/interactivechat-show-items-inventory-in-chat-custom-chat-keywords-bungee-velocity-support.7587/

**Comment by mfnalex** [comment]
![image](https://user-images.githubusercontent.com/1122571/173599909-13b2b6bf-f6ab-48a4-867b-cf0f86fcb363.png)
And it was apparently removed from Spigot

**Comment by mfnalex** [comment]
I don't think this is useful. Also it adds a ton of dependencies we actually don't need for anything else. But yeah, let's vote. I'm against it

**Comment by KaspianDev** [comment]
it needs ProtocolLib, PlaceholderAPI, Vault which are pretty standard

Talking about PAPI, we can use Statistic addon for some cool stats in the hover popup

**Comment by cxlina** [comment]
I don't mind it, but also don't really need it.

**Comment by KaspianDev** [comment]
#3
